Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volumegroup-snapshot.md

```diff
@@ -0,0 +1,107 @@
+---
+title: VolumeGroup Snapshots
+---
+
```

Comment:
Can you provide an intro paragraph to describe what volumegroup snapshots are, and the main scenario(s) where they are useful?